_data/api-tools.yml

```diff
@@ -502,3 +502,12 @@
   runtime: "https://img.shields.io/badge/API_Runtime-No-red.svg"
   testing: "https://img.shields.io/badge/API_Testing-Yes-brightgreen.svg"
   notes: "<br/><em>Note: Use Curl-like command or OpenAPI contract to perform API scan. An existing Github Action can perform a scan automatically before deployment.</em>"
+- name: "ZeroThreat"
+  link: "https://zerothreat.ai/solutions/api-security-testing"
+  from: "ZeroThreat"
+  license: "https://img.shields.io/badge/license-Commercial-9cf.svg"
+  platforms: "https://img.shields.io/badge/Platform-SaaS%2C On--Prem-inactive.svg"
+  posture: "https://img.shields.io/badge/API_Posture-Yes-brightgreen.svg"
+  runtime: "https://img.shields.io/badge/API_Runtime-Yes-brightgreen.svg"
+  testing: "https://img.shields.io/badge/API_Testing-Yes-brightgreen.svg"
```

Comment:
Are you sure it really covers all three?

Reply:
> Are you sure it really covers all three?

Yes.